src/scancode/cli.py

```diff
@@ -479,6 +479,8 @@ def fixed_width_file_name(path, max_length=25):
         return ''
 
     filename = fileutils.file_name(path)
+    terminal_width = click.get_terminal_size()[0]
```

Comment:
I think that the terminal size should be computed outside of the function. Otherwise you are making an extra call on each file for no good reason. It is safe to assume that we do not support resizing terminals while a scan runs.
Also it is cleaner and more explicit to use
`terminal_width, _terminal_height = click.get_terminal_size()`

And you could add back the max_length parameter to the func sig.... defaulting to a TERM_WIDTH constant computed  may be this way:
`TERM_WIDTH, _ = click.get_terminal_size()`


this allow you to actually test the code and keep all the tests.... Removing the tests does not make sense IMHO.